index.bs

```diff
@@ -1299,6 +1299,38 @@ Verifiable Random Selection Procedure</h5>
 			The incomplete terms are assigned in result order.
 	</ol>
 
+<h5 id="AB-TAG-recall">
+Recalling AB or TAG members</h5>
+
+	The [=Advisory Committee=] <em class=rfc2119>may</em>
+	<dfn export>recall</dfn> participants of the [=AB=] or [=TAG=]
+	(whether [[#AB-TAG-elections|elected]] or [[#TAG-appointments|appointed]]).
+
+	An [=Advisory Committee representative=] initiates a [=recall=]
+	by sending a request to the Team, and <em class=rfc2119>should</em> also share this request with the Advisory Committee.
+	The request <em class=rfc2119>should</em> say “I call for removal of (an) AB and/or TAG participant(s)”,
+	<em class=rfc2119>must</em> identify the [=AB=] or [=TAG=] [=participant=] or [=participants=] targeted,
+	and <em class=rfc2119>should</em> also include the rationale for the proposed [=recall=].
+
+	Messages sent by [=Advisory Committee representatives=] to the [=Advisory Committee=]
+	with the ostensible intent of garnering support for a recall that has not yet been formally requested
+	<em class=rfc2119>should</em> also be treated by the [=Team=]
+	as a request to initiate the recall.
+
+	Within one week, the [=Team=] <em class=rfc2119>must</em> initiate a [=Advisory Committee Override=]
+	on the recall proposal.
+
+	If the [=Advisory Committee Override=] proposal passes,
+	the seat(s) of the relevant [=AB=] or [=TAG=] participant(s) are vacated immediately.
+
+	The conclusion of the [=Advisory Committee Override=] is final,
+	and cannot be the subject of a [=Formal Objection=] nor of an [=AC Appeal=].
+
+	A single [=Member=] or group of [=related Members=] cannot re-invoke this process
+	sooner than 6-month since their previous invocation,
```

Comment:
It's unclear when the six-month timer begins. So, this —
```suggestion
	A single [=Member=] or group of [=related Members=] cannot re-invoke the [=recall=] process
	less than six months following the conclusion of their previous invocation,
```

Reply:
— or this —
```suggestion
	A single [=Member=] or group of [=related Members=] cannot re-invoke the [=recall=] process
	less than six months following the initiation of their previous invocation,
```

Reply:
I meant the second one, but it seems to me that the existing phrasing already is unambiguous (and shorter). Can you help me understand why you think it's not clear?

Reply:
> why you think it's not clear

Uh... because it wasn't clear to me, after reading several times. Hence my comment, and alternate suggestions.

It matters, because if the process initiated on 2025-06-01 takes 2025-09-15, re-invocation request might not be accepted until 2025-12-01 *or* until 2026-03-15. It's important to be able to calculate which date applies.

Reply:
I think the key to me is that "invoking" is a point in time, at the start, not a period, just the same way "initiating is" (with the nuance that invoking is more specific: it's initiating by saying something).

So, your second phrasing matches what I meant, but to me it feels redundant: at the since the start of the beginning of the initiation of the invocation… ;)

Reply:
I think that your phrasing is ambiguous enough that it could be read as starting the six month timer at conclusion of the previous invocation, which would be certain to prevent overlapping recall process runs, which seems a likely reason for this timer. 

Starting the timer at "invocation time" leaves the potential of overlap, if an early run takes longer than 6 months (which seems possible if unlikely).

It might help to include a reason for this required time lapse?
```suggestion
	A single [=Member=] or group of [=related Members=] cannot re-invoke the [=recall=] process
	less than six months following the start of the process based on their previous invocation,
```

Reply:
> It might help to include a reason for this required time lapse?

Generally, if you ask for a recall and the motion does not pass, being able to ask again and again has more to do with harassment than with accountability, even if you target different subsets of the elected body every time, so we should try to prevent it. However, if new facts come to light, asking for another recall might be appropriate.

> if an early run takes longer than 6 months (which seems possible if unlikely).

I don't think it is or should be possible:
- first someone invokes the recall
- Within one week, the Team must open a poll to see if there's support
- Withing one week of that, if we don't reach 5%, it's over, and if we do, we launch the full recall vote
- after the vote, the procedure ends, and there's no recourse

It is true that the Process doesn't specify how long the vote should stay open, but a 6 month balloting period is unheard of and would be unreasonable. Maybe we should encode into the Process that the ballot stay open for 28 days, which is a typical duration for that sort of things (e.g., that's what was used for the [EME AC Appeal](https://www.w3.org/2002/09/wbs/33280/eme-appeal-vote/results))